content/linux-gaming.md

```diff
@@ -155,7 +193,7 @@ Open the <u>Pop!_Shop</u> application then either search for GameHub, or navigat
 
 ![Pop!_Shop GameHub](/images/linux-gaming/pop-shop_gamehub.png)
 
-Once installed, use the Activities Overview to search for and run <u>GameHub</u>.
+Once installed, use the Application button to search for and run <u>GameHub</u>.
```

Comment:
The button says "Applications", so I'm thinking Application should probably be plural to match the button, right?